packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -34,7 +33,7 @@ type TreeNodeProps = ComponentPropsWithoutRef<typeof Tree.Item>;
 /** @alpha */
 export interface TreeNodeRendererOwnProps {
   /** Node that is rendered. */
-  node: FlatTreeNode;
+  node: PresentationHierarchyNode;
```

Comment:
Is there a reason to move item position within the tree outside of the node? Do you think it would make the API more flexible?

Reply:
I wanted to keep virtualization related code in one place. I don't think it changes API flexibility.

Reply:
We should probably discuss this later when we discuss the whole api as we seem to keep jumping from this to node, back to this.